docs/source/conf.py

```diff
@@ -226,6 +226,8 @@
     "https://opensource.org/license/apache2-0-php/",
     "https://docs.github.com/en/rest/overview/other-authentication-methods#via-username-and-password",
     "https://docs.snowflake.com/en/developer-guide/snowpark/reference/python/api/snowflake.snowpark.DataFrameWriter.saveAsTable.html"
+    # temporarily, until the file actually exists in `main`
+    "https://github.com/kedro-org/kedro/blob/main/kedro/templates/project/%7B%7B%20cookiecutter.repo_name%20%7D%7D/.flake8",
```

Comment:
Apparently this has no effect in preventing the warning (then turned into an error) 🤔 Will ignore it for now